schema/tables/disk_info.yml

```
@@ -0,0 +1,26 @@
+name: disk_info
+platforms:
+  - chrome
```

Since this is an existing table in osquery, we'll need to include the original platform value. otherwise, this will override this value completely. 

```suggestion
platforms:
  - windows
  - chrome
```